source/elements/oneDNN/source/graph/ops/activation/Clamp_1.rst

```diff
@@ -0,0 +1,66 @@
+.. SPDX-FileCopyrightText: 2020-2021 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+-----
+Clamp
+-----
+
+**Versioned name**: *Clamp-1*
+
+**Category**: *Activation*
+
+**Short description**: *Clamp* operation represents clipping activation
+function.
+
+**OpenVINO description**: This OP is as same as `OpenVINO OP
```

Comment:
Just want to confirm whether it's good to put openvino link here.

Reply:
Not a fan. Will remove those as I push the unified style of operation specification (the one you just completed, thanks BTW :-)).

Reply:
removed all the OV links with new style.